plugins/koterm.koplugin/profile

```diff
@@ -0,0 +1,27 @@
+# This file gets executed on every start of the KOTerm shell.
```

Comment:
Adding a shebang will make shellcheck pick it up.
```suggestion
#!/bin/sh
# This file gets executed on every start of the KOTerm shell.
```

It will then suggest to separate assignment and exporting, cf.
https://github.com/koreader/koreader/blob/1d409496992711b7473820426baa1b27ff208ae2/platform/kobo/koreader.sh#L240-L241